### PR TITLE
Varun_Assignment_9

### PR DESCRIPTION
**Pull request by varunj822**
image

### CODE REVIEW

File: src/varun/Assignment_9.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+public class Assignment_9 {
+	int maggieStock = 50;
+	int dosaStock = 43;
+	int pouchesStock = 39;
+	int panipuriStocks = 43;
+	int masalaStock = 73;
+
+	void stockOfItemsAvlbl(String itemName, int itemQuantity) {
+		if(itemName.equalsIgnoreCase("Maggie")) {
+			maggieStock = maggieStock-itemQuantity;
```
Comment (harshvegada) [inline src/varun/Assignment_9.java:12]: Here first you should check whether we have enough stock that fulfilled the current requirement of customer?

File: src/varun/Assignment_9.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+			if(maggieStock<0) {
+				outOfStock(itemName);
+			}
+			else if(maggieStock<=10)
+			System.out.println(itemName+ " is running out of stock with only " +maggieStock+ " quantity remaining");
```
Comment (harshvegada) [inline src/varun/Assignment_9.java:17]: why are saying if stock is less than 10 then stock is not available ?

**Review by harshvegada — CHANGES_REQUESTED** [review]
Check requirement again and update this

Reference 
https://github.com/KrishnaKTechnocredits/JAVATechnoJuly2021/pull/377/files

File: src/varun/Assignment_9.java (hunk 2 above)
Comment (harshvegada) [reply src/varun/Assignment_9.java:17]: Yes, similar to OOS
OK

**Comment by varunj822** [comment]
image

**Review by harshvegada — APPROVED** [review]
Code looks good to me,
 Good to merge!

**Pull request merged**